features/docs/formatters/event_stream_formatter.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Event Stream Formatter
+
+  Emits a stream of events compatible with https://github.com/cucumber/cucumber-pretty-formatter
+
+  Background:
+    Given the standard step definitions
+
+  Scenario: A simple scenario
+    Given a file named "features/simple.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given this step passes
+      """
+    When I run `cucumber -q --format Cucumber::Formatter::EventStream`
+    Then it should pass
+    And the stdout should contain exactly:
+      """
+      {"event":"TestRunStarted","protocol_version":"0.0.1"}
+      {"event":"GherkinSourceRead","id":"features/simple.feature:1","source":"Feature:\n  Scenario:\n    Given this step passes"}
+      {"event":"StepDefinitionFound","source_id":"features/simple.feature:3","definition_id": "features/step_definitions/steps.rb:1"}
```

Comment:
Could this be named `StepDefinitionMatched` instead? I think *found* can possibly be mistaken with *loaded*.